modules/cloud-run/README.md

```diff
@@ -0,0 +1,166 @@
+# Cloud Run Module
+
+Cloud Run management, with support for IAM roles and optional Eventarc trigger creation.
+
+## Examples
+
+### Traffic split
+
+This deploys a Cloud Run service with traffic split between two revisions.
+
+```hcl
+module "cloud_run" {
+  source     = "../../modules/cloud-run"
+  project_id = "my-project"
+  name       = "hello"
+  revision_name = "green"
+  containers = [{
+    image   = "us-docker.pkg.dev/cloudrun/container/hello"
+    command = null
+    args    = null
+    env     = null
+    env_from = null
+    ports = null
+    resources = null
+    volume_mounts = null
+  }]
+  traffic = {
+    "blue" = 25
+    "green" = 75
+  }
+}
+# tftest:skip
+```
+
+### Eventarc trigger (Pub/Sub)
+
+This deploys a Cloud Run service that will be triggered when messages are published to Pub/Sub topics.
+
+```hcl
+module "cloud_run" {
+  source     = "../../modules/cloud-run"
+  project_id = "my-project"
+  name       = "hello"
+  containers = [{
+    image   = "us-docker.pkg.dev/cloudrun/container/hello"
+    command = null
+    args    = null
+    env     = null
+    env_from = null
+    ports = null
+    resources = null
+    volume_mounts = null
+  }]
+  pub_sub_triggers = [
+    "topic1",
+    "topic2"
+  ]
+}
+# tftest:skip
+```
+
+### Eventarc trigger (Audit logs)
+
+This deploys a Cloud Run service that will be triggered when specific log events are written to Google Cloud audit logs.
+
+module "cloud_run" {
```

Comment:
please add quotes, for formatting and example testing

Reply:
Done